### ISSUE TITLE
Remove tab stop/keyboard focus on the "HTML" cards in HTML report

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#3421]
## Describe the bug

The HTML cards in failure instances in the HTML report are currently focusable. But these cards need not be selectable/focusable as there is no action available(non interactive) once we focus on them. 

Removing the keyboard focus would not have an impact on AT users as the table content is available in browser/scan mode using table commands

**To Reproduce**

1. Generate a HTML report on a page with failures
2. Tab through the report to focus on the HTML cards

## Expected behavior
Remove keyboard focus on the HTML card

## Screenshots

![image](https://user-images.githubusercontent.com/22068434/94736336-ed4cc400-0320-11eb-88e4-6aa6232112a9.png)

## Context 
- OS Name: Windows 10, Version: 1903, Build 18362.476
- AI-Web Version & Environment: Production 2.21.1 
- Browser Version: Google Chrome 85.0.4183.121
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

**Comment by RobGallo** [comment]
While not an exact duplicate, issue #1796  seems to encompass the issue reported in this bug. So we will label this as a duplicate and address the problem in the earlier bug.

@iamrafan , if you see that as a problem, please let us know.
